### PR TITLE
update video miner metric to minutes transcoded instead of activation cost

### LINKED ISSUE #80: Fix cost to activate node number on video miners page
https://livepeer.org/video-miners shows the cost to activate a node as free, but this isn't quite true and has caused some confusion amongst node operators since activation for an orchestrator does require ETH transaction fees. Maybe there is an argument for saying its free if you're running a transcoder as a part of a pool, but that distinction is not made here. The wording here should be updated so that at the very least users do not think that activating an orchestrator is completely free.

### PR DESCRIPTION
**Pull request by adamsoffer**
Closes #80

### CODE REVIEW

File: public/locales/zh/video-miners.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "page-video-miners-numbers-fees": "28.5 ETH",
   "page-video-miners-numbers-fees-text": "赚取的总费用",
   "page-video-miners-numbers-cost": "免费",
-  "page-video-miners-numbers-cost-text": "运行节点的成本",
+  "page-video-miners-numbers-cost-text": "总分钟转码视频",
```
Comment (yondonfu) [inline public/locales/zh/video-miners.json:14]: This is great. I just learned how to say transcoding in Chinese.

File: lib/document-helpers.ts [hunk 2]
```diff
@@ -143,3 +143,9 @@ export {
   getTotalVolume,
   getTotalActiveNodes,
 };
+
+export const getTotalMinutes = async () => {
+  const response = await fetch("https://explorer.livepeer.org/api/usage");
```
Comment (yondonfu) [inline lib/document-helpers.ts:148]: Is this endpoint new and where is it served from?

**Review by yondonfu — APPROVED** [review]
LGTM, but just had a few clarifying questions.

File: lib/document-helpers.ts (hunk 2 above)
Comment (adamsoffer) [reply lib/document-helpers.ts:148]: This is the same data that powers the charts on the explorer. What's new is the API endpoint that now exposes this data.

Comment (adamsoffer) [reply lib/document-helpers.ts:148]: https://github.com/livepeer/livepeerjs/pull/956/files

Comment (yondonfu) [reply lib/document-helpers.ts:148]: Nice 👍

**Pull request merged**